test/constructors.jl

```diff
@@ -18,8 +18,6 @@ module TestConstructors
 
     @test isequal(dt, DataTable(Any[NullableCategoricalVector(zeros(3)),
                                     NullableCategoricalVector(ones(3))]))
-    @test isequal(dt, DataTable(x1 = [0.0, 0.0, 0.0],
```

Comment:
This test still applies: it checks that comparisons work across types. You can convert it to `NullableArray` like below.